### DISCUSSION TITLE
Error of Shared Object When Running Dosxyznrc Examples

### DISCUSSION THREAD

**Discussion by ROOTRT** [nrc-cnrc/EGSnrc#754] [Questions] [answered]
- **Description**

The error occured when running dosxyznrc examples via dosxyznrc GUI
![image](https://user-images.githubusercontent.com/87711551/126598123-fcaff050-0bfd-471c-b06e-6d90d22bc7c8.png)
All input files in BEAM_EX10MeVe, BEAM_EX16MVp, BEAM_EXphantom, BEAM_EXslabs occured this error when selecting files with .egsinp suffix.

- **Environment**

The EGSnrc toolkit was compiled following the official instruction strictly via the command line procedure. The logs are attached as follows.
configure-linux.log
configure-linux-phresher.log

The version of EGSnrc is v2021.

We look forward to your reply and request for a list of compatible building tools.

Thanks in advance!

**Comment by rtownson** [comment, marked as answer]
It sounds like you are trying to open a BEAMnrc input file in dosxyznrc_gui, which can only open DOSXYZnrc input files. Try using beamnrc_gui instead.

**Comment by ROOTRT** [comment]
Thank you so much!  The mistake is so simple but I did't find. My life saver.
